HW2.ipynb

```diff
@@ -0,0 +1,94 @@
+# **Reference (source)**: 
+[IBM Qiskit Lab](lhttps://lab.quantum-computing.ibm.com)
+
+  {
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9c9a22bc-1315-4571-96cd-ebc368eb94f1",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# **Task**\n",
+    "\n",
+    "I would like to make a quantum circuit that performs the addition of 1+1= 2 .\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "75820028-88b3-42f0-b4e1-736a2772a46c",
+   "metadata": {},
+   "source": [
+    "# **How ?**\n",
+    " I am not sure. But may be doing their own research and practices."
```

Comment:
Have a look at https://github.com/ubsuny/QC_fullAdder/blob/6217a24f6fd1b897eaec1c86a225a14f62c3be14/Homework%203%20Full%20adder.ipynb
for an example how the documentation should look like